data/en/bots/462212977882562571.md

```diff
@@ -0,0 +1,17 @@
+---
+avatar: >-
+  https://images-ext-1.discordapp.net/external/rEC-FKA-1pKc_0Vo_6yetpfcdRsNLQr__4jYtqusZLg/%3Fsize%3D2048/https/cdn.discordapp.com/avatars/462212977882562571/cef0139e279e65dfe5ddf7cd2675cfac.png
+cover: null
+custom_path: mardatonne
+description: This bot will let you do whatever you want
+github:
+  owner: NeotiDev
+images: []
+link: >-
+  https://discordapp.com/oauth2/authorize?client_id=462212977882562571&scope=bot&permissions=8
+nsfw: false
+pagename: Mardatonne
+prefix: +
+support: NqMA6xC
```

Comment:
You should provide the full link, like `https://discord.gg/NqMA6xC`
The site doesn't auto-add the link to the code